src/App.js

```diff
@@ -1,16 +1,36 @@
 import React from "react";
-import logo from "./logo.png";
 import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import About from "./Components/About";
+import Education from "./Components/Education";
+import Menu from "./Components/Navbar";
+import Hero from "./Components/Hero";
+import Project from "./Components/Project";
+import Work from "./Components/WorkExpereince";
+import Contact from "./Components/Contact";
+import Skills from "./Components/skills";
```

Comment:
```suggestion
import Skills from "./Components/Skills";
```

Components should be capital letter